src/tictactoe.js

```diff
@@ -0,0 +1,112 @@
+function Square(props) {
```

Comment:
Do we need this file at all?

Reply:
Keep for now, as it is a useful reference?